Description: force ld to not cache symbols tables.
 This patch, wich should be considered Debian-only (in other words, it should
 not be pushed upstream), forces ld to not cache the symbol tables of input
 files in memory to avoid memory exhaustion during the linking phase.
 .
 This might make linking slower but reducing the overall memory consumption.
Author: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Forwarded: not-needed
Last-Update: 2014-03-05

---
 Tools/qmake/mkspecs/features/unix/default_post.prf |    3 +++
 1 file changed, 3 insertions(+)

--- a/Tools/qmake/mkspecs/features/unix/default_post.prf
+++ b/Tools/qmake/mkspecs/features/unix/default_post.prf
@@ -69,3 +69,6 @@
 }
 
 load(default_post)
+
+# Make ld not cache the symbol tables of input files in memory to avoid memory exhaustion during the linking phase.
+QMAKE_LFLAGS += -Wl,--no-keep-memory
